CONTRIBUTING.md

```diff
@@ -1,3 +1,35 @@
+* [Welcome!](#welcome)
+   * [HTTPS Everywhere Source Code Layout](#https-everywhere-source-code-layout)
+   * [Submitting Changes](#submitting-changes)
+* [Contributing Rulesets](#contributing-rulesets)
+   * [General Info](#general-info)
+   * [New Rulesets](#new-rulesets)
+   * [Minimum Requirements for a Ruleset PR](#minimum-requirements-for-a-ruleset-pr)
+   * [Testing](#testing)
+   * [Ruleset Style Guide](#ruleset-style-guide)
+      * [Motivation](#motivation)
+      * [Indentation &amp; Misc Stylistic Conventions](#indentation--misc-stylistic-conventions)
+      * [Wildcards in Targets](#wildcards-in-targets)
+         * [Left-Wildcards](#left-wildcards)
+         * [Edge-Case: Right-Wildcards](#edge-case-right-wildcards)
+      * [Complicated Regex in Rules](#complicated-regex-in-rules)
+      * [Enumerating Subdomains](#enumerating-subdomains)
+      * [Target Ordering](#target-ordering)
+      * [Rule Ordering](#rule-ordering)
+      * [Non-working hosts](#non-working-hosts)
+      * [Ruleset Names](#ruleset-names)
+         * [Filenames](#filenames)
```

Comment:
Is the `Filenames` section actually supposed to be nested in the `Ruleset Names` section?

Reply:
I think so, yeah.